schema.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
       required: ['createdAt']
     }
   },
-  required: ['id', 'correlationId', 'meta', 'ack', 'payload'],
+  required: ['id', 'correlationId', 'meta'],
```

Comment:
That is not true. In the [current master](https://github.com/tcdl/msb/blob/e4fe16f04e1ba30a3b859e1670ccf89a52cf31c4/schema.js#L22) meta can only be an `object`.